common/lib/xmodule/xmodule/library_tools.py

```diff
@@ -21,14 +21,17 @@ def _get_library(self, library_key):
         Given a library key like "library-v1:ProblemX+PR0B", return the
         'library' XBlock with meta-information about the library.
 
+        A specific version may be specified.
+
         Returns None on error.
         """
         if not isinstance(library_key, LibraryLocator):
             library_key = LibraryLocator.from_string(library_key)
-        assert library_key.version_guid is None
```

Comment:
Optional: You could use `@contract` to specify that `library_key.version_guid` isn't `None`.